scmailclient/scmail.py

```diff
@@ -0,0 +1,493 @@
+import click
+import pprint
+import re
+from pathlib import Path
+from os import environ
+import getpass
+import logging
+import time
+import requests
+
+
+from gpgopt import GpgOpt
+
+
+# Default to localhost if url is not given
+SECUREMAILBOX_URL = environ.get(
+    "SECUREMAILBOX_URL", "http://securemailbox.duckdns.org:8082/"
+)
+
+# Logging Level
+LOGGING_LEVEL = environ.get("LOGGING_LEVEL", logging.DEBUG)
+
+
+def check_fingerprint(ctx, param, value):
```

Comment:
Click recommend the 3 arguments format and it will give a warning if only two arguments appear (although one of them is not used). In this case, I considered let `param` as a parameter. 
In theory, it is not required. The program will run normally except for the warning.